### PR TITLE
Do not build dependencies if they are found to be installed

### PR DESCRIPTION
**Pull request by jnumm**
Would be useful for the Debian Packagers. 

(Cc: @tillea, @pjotrp)

### CODE REVIEW

**Comment by pjotrp** [comment]
Thank you! CI fails to find the tabix.hpp. Any idea?

File: meson.build [hunk 1]
```diff
@@ -56,15 +55,77 @@ else
...
+    tabixpp_lib = static_library('custom_tabixpp',
+                                 tabixpp_src,
+                                 include_directories : tabixpp_inc)
+    tabixpp_dep = declare_dependency(link_with : tabixpp_lib,
+                                     include_directories : tabixpp_inc)
+endif
+if not vcflib_dep.found()
+    vcflib_lib = static_library('custom_vcflib',
+                                vcflib_src,
+                                include_directories : vcflib_inc)
```
Comment (jnumm) [inline meson.build:111]: vcflib uses tabix, so this is supposed to be include_directories : [vcflib_inc, tabixpp_inc]

**Comment by jnumm** [comment]
Ci checks passing now.

**Comment by pjotrp** [comment]
Thank you!

**Pull request merged**